lib/InstallerUtils.py

```diff
@@ -81,7 +81,10 @@ def get_product_catalog(config, all_products=False):
 
     for item in all_product_data:
         val = all_product_data[item]
-        all_product_data[item] = yaml.safe_load(val)
+        try:
```

Comment:
I was hitting an AttributeError exception on the yaml.safe_load here.  It's in the product catalog as yaml, but it seems to get converted from yaml when the `json.loads` is called.